services/black/src/api/artiso.AdsdHotel.Black.Api.csproj

```diff
@@ -7,18 +7,26 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Tye.Extensions.Configuration" Version="0.4.0-alpha.20371.1" />
-    <PackageReference Include="NServiceBus" Version="7.4.3" />
+    <PackageReference Include="MongoDB.Driver" Version="2.11.3" />
+    <PackageReference Include="NServiceBus" Version="7.4.4" />
     <PackageReference Include="NServiceBus.Callbacks" Version="3.0.0" />
     <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="2.2.0" />
     <PackageReference Include="NServiceBus.RabbitMQ" Version="6.0.0" />
     <PackageReference Include="NServiceBus.Extensions.DependencyInjection" Version="1.*" />
     <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.*" />
-    <PackageReference Include="NServiceBus.MicrosoftLogging.Hosting" Version="1.*" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\infrastructure\src\dataStorage\artiso.AdsdHotel.Infrastructure.DataStorage.csproj" />
+    <ProjectReference Include="..\..\..\infrastructure\src\nservicebus\artiso.AdsdHotel.Infrastructure.NServiceBus.csproj" />
     <ProjectReference Include="..\commands\artiso.AdsdHotel.Black.Commands.csproj" />
     <ProjectReference Include="..\events\artiso.AdsdHotel.Black.Events.csproj" />
+    <ProjectReference Include="..\..\..\infrastructure\src\mongoDataStorage\artiso.AdsdHotel.Infrastructure.MongoDataStorage.csproj" />
   </ItemGroup>
+	<ItemGroup>
+		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+			<_Parameter1>artiso.AdsdHotel.Black.Tests</_Parameter1>
```

Comment:
_tabs_

Reply:
Don't you like spaces???? I just copy/pasted this^^ Ok, tabs it is.

Reply:
no... I meant: put spaces instead of tabs 😄

Reply:
Oo that's a discussion for itself^^ Basically I don't care, but we should bring that up.